README.md

```diff
@@ -117,6 +117,19 @@ Kubernetes Release | DigitalOcean CSI Driver Version
 
 The [DigitalOcean Kubernetes](https://www.digitalocean.com/products/kubernetes/) product comes with the CSI driver pre-installed and no further steps are required.
 
+---
+**Driver modes:**
+
+By default, driver runs in both controller and node mode. It would create disk Volumes on DigitalOcean infrastructure and mount them on the required node.
```

Comment:
```suggestion
By default, the driver supports both the [controller and node mode](https://kubernetes-csi.github.io/docs/deploying.html): it can manage DigitalOcean Volumes via the cloud API and mount them on the required node. The actually used mode is determined by how the driver is deployed and configured.
```